addons/sale/data/sale_demo.xml

```diff
@@ -548,6 +584,45 @@
             <field name="price_unit">12.50</field>
         </record>
 
+        <record id="sale_order_19" model="sale.order">
+            <field name="partner_id" ref="base.res_partner_3"/>
+            <field name="partner_invoice_id" ref="base.res_partner_address_25"/>
+            <field name="partner_shipping_id" ref="base.res_partner_address_25"/>
+            <field name="user_id" ref="base.user_demo"/>
+            <field name="pricelist_id" ref="product.list0"/>
+            <field name="team_id" ref="sales_team.team_sales_department"/>
+        </record>
+
+        <record id="sale_order_line_42" model="sale.order.line">
```

Comment:
rename those to sale_order_19_...

Reply:
It would be better to reuse an existing demo data instead of creating a new one.